### PR TITLE
Remove mount server

### PR DESCRIPTION
**Pull request by smalyala**
Removes
- code
- Go tests
- releases/deployments
- CI tests

Jira: [INT-1197]

[INT-1197]: https://pachyderm.atlassian.net/browse/INT-1197?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/server/pfs/fuse/loopback.go [hunk 1]
```diff
@@ -35,12 +35,6 @@ const (
 	dirty                  // we have full content for this file and the user has written to it
 )
 
-func (l *loopbackRoot) setState(mountName, state string) {
-	l.mu.Lock()
-	defer l.mu.Unlock()
-	l.stateMap[mountName] = state
```
Comment (msteffen) [inline src/server/pfs/fuse/loopback.go:41]: Is this just dead code? What's the connection between this function and the mount server?

**Review by msteffen — APPROVED** [review]
One question, but overall LGTM

File: src/server/pfs/fuse/loopback.go (hunk 1 above)
Comment (smalyala) [reply src/server/pfs/fuse/loopback.go:41]: That's correct. It's no longer used. It's only caller was in the mount server code.

**Review by molinamelendezj — APPROVED** [review]
will this be for `2.10.0`? will it get a doc announcement; the helm chart has no reference to mount server image right? I think its used with a helm value of `extraContainers:`

File: src/server/pfs/fuse/BUILD.bazel [hunk 2]
```diff
@@ -59,17 +49,14 @@ go_test(
...
         "//src/internal/client",
         "//src/internal/dockertestenv",
         "//src/internal/errors",
         "//src/internal/pctx",
         "//src/internal/require",
         "//src/internal/testpachd/realenv",
-        "//src/internal/testutil",
         "//src/internal/testutil/random",
         "//src/pfs",
-        "//src/pps",
```
Comment (molinamelendezj) [inline src/server/pfs/fuse/BUILD.bazel:72]: how come pps is getting removed here?

Comment (smalyala) [reply src/server/pfs/fuse/BUILD.bazel:72]: After removing mount server code, that dependency is no longer used in the fuse package.

**Comment by smalyala** [comment]
Yes, 2.10.0. With the Jupyter fuse-less changes going out in 2.9, the docs should be updated such that mount server is no longer needed nor referenced in the docs. I'll confirm with LB that this is the case for 2.10.0.

**Pull request merged**